doc/Architecture/arc-tec_x-road_technologies.md

```diff
@@ -205,14 +213,14 @@ Note: OP-monitoring daemon is an additional component of the X-Road.
 <a id="Ref_Technology_matrix_of_the_operational_monitoring_daemon" class="anchor"></a>
 Table 5. Technology matrix of the operational monitoring daemon
 
-Technology           | Op. Mon.<br/>Daemon Main | Op. Mon.<br/>Database | Op. Mon.<br/>Service | Configuration<br/>Client
-:------------------- | :----------------------: | :-------------------: | :------------------: | :---:
-Java 11              | X                        | X                     | X                    | X
-Logback              | X                        | X                     | X                    | X
-Akka 2               | X                        | X                     |                      |
-PostgreSQL 10        | X                        | X                     |                      |
-Liquibase 3          | X                        | X                     |                      |
-Dropwizard Metrics 4 | X                        | X                     |                      |
-systemd              | X                        |                       |                      | X
+| Technology           | Op. Mon.<br/>Daemon Main | Op. Mon.<br/>Database | Op. Mon.<br/>Service | Configuration<br/>Client |
+|:---------------------|:------------------------:|:---------------------:|:--------------------:|:------------------------:|
+| Java 11              |            X             |           X           |          X           |            X             |
+| Logback              |            X             |           X           |          X           |            X             |
+| Akka 2               |            X             |           X           |                      |                          |
+| PostgreSQL 10        |            X             |           X           |                      |                          |
```

Comment:
Should PostgreSQL 12 (Ubuntu 20.04) and PostgreSQL 14 (Ubuntu 22.04) be added? Also, PostgreSQL versions used by RHEL7 and RHEL8 should be checked.